src/main/scala/org/broadinstitute/dsde/workbench/sam/schema/JndiSchemaDAO.scala

```diff
@@ -256,6 +258,8 @@ class JndiSchemaDAO(protected val directoryConfig: DirectoryConfig, val schemaLo
 
     val may = new BasicAttribute("MAY")
     may.add(Attr.proxyEmail)
+    may.add(Attr.googleSubjectId)
+
```

Comment:
Why is it optional?

Reply:
@dvoet for invite endpoint, we don't know the invited user's googlesubjectId yet when it's created